misc/config_tools/schema/types.xsd

```diff
@@ -262,9 +305,8 @@ As :option:`vm.clos.vcpu_clos` specifies the index of the CLOS to be associated
 <xs:complexType name="SSRAMInfo">
   <xs:sequence>
     <xs:element name="SSRAM_ENABLED" type="Boolean" default="n">
-      <xs:annotation>
-        <xs:documentation>Enable PTCM (Platform Tuning Configuration
-Manager).</xs:documentation>
+      <xs:annotation acrn:title="Software SRAM (for real-time apps)">
```

Comment:
I think we can drop mentioning "(for real-time apps)" in the title since it's in the description and it makes the title long.